language/automl/automl_natural_language_dataset.py

```diff
@@ -0,0 +1,300 @@
+
+    # Get the full path of the dataset.
+    dataset_full_id = client.dataset_path(
+        project_id, compute_region, dataset_id
+    )
+
+    # Set the output URI
+    output_config = {"gcs_destination": {"output_uri_prefix": output_uri}}
+
+    # Export the data to the output URI.
+    response = client.export_data(dataset_full_id, output_config)
+
+    print("Processing export...")
+    # synchronous check of operation status.
+    print("Data exported. {}".format(response.result()))
+
+
+# [END automl_natural_language_export_data]
+
+
+# [START automl_natural_language_delete_dataset]
+def delete_dataset(project_id, compute_region, dataset_id):
+    """Delete a dataset.
+    Args:
+        project_id: Id of the project.
+        compute_region: Region name.
+        dataset_id: Id of the dataset.
+    """
+    client = automl.AutoMlClient()
+
+    # Get the full path of the dataset.
+    dataset_full_id = client.dataset_path(
+        project_id, compute_region, dataset_id
+    )
+
+    # Delete a dataset.
+    response = client.delete_dataset(dataset_full_id)
+
+    # synchronous check of operation status.
+    print("Dataset deleted. {}".format(response.result()))
+
+
+# [END automl_natural_language_delete_dataset]
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    subparsers = parser.add_subparsers(dest="command")
+
+    create_dataset_parser = subparsers.add_parser(
+        "create_dataset", help=create_dataset.__doc__
+    )
+    create_dataset_parser.add_argument("dataset_name")
+    create_dataset_parser.add_argument(
+        "multilabel", nargs="?", choices=["False", "True"], default="False"
+    )
+
+    list_datasets_parser = subparsers.add_parser(
+        "list_datasets", help=list_datasets.__doc__
+    )
+    list_datasets_parser.add_argument(
+        "filter_", nargs="?", default="text_classification_dataset_metadata:*"
+    )
+
+    get_dataset_parser = subparsers.add_parser(
+        "get_dataset", help=get_dataset.__doc__
+    )
+    get_dataset_parser.add_argument("dataset_id")
+
+    import_data_parser = subparsers.add_parser(
+        "import_data", help=import_data.__doc__
+    )
+    import_data_parser.add_argument("dataset_id")
+    import_data_parser.add_argument("path")
+
+    export_data_parser = subparsers.add_parser(
+        "export_data", help=export_data.__doc__
+    )
+    export_data_parser.add_argument("dataset_id")
+    export_data_parser.add_argument("output_uri")
+
+    delete_dataset_parser = subparsers.add_parser(
+        "delete_dataset", help=delete_dataset.__doc__
+    )
+    delete_dataset_parser.add_argument("dataset_id")
+
+    project_id = os.environ["PROJECT_ID"]
+    compute_region = os.environ["REGION_NAME"]
+
+    args = parser.parse_args()
+
+    if args.command == "create_dataset":
+        multilabel = True if args.multilabel == "True" else False

```

Comment:
This line can be simplified to:
multilabel = (args.multilabel == "True")

Reply:
doesn't affect snippet quality or functionality. won't fix.